src/plugins/sync_to_async.ts

```diff
@@ -1,5 +1,5 @@
 import * as types from "babel-types";
-import { compact, uniq, chain, get } from "lodash";
+import { compact, uniq, chain, get, last } from "lodash";
```

Comment:
is `last` needed here?

Reply:
Nope, I refactored out the use of `last`, will clean up